src/diffusers/loaders.py

```diff
@@ -1073,6 +1136,17 @@ def load_lora_into_text_encoder(cls, state_dict, network_alpha, text_encoder, lo
                             f"{name}.out_proj.lora_linear_layer.down.weight"
                         ] = text_encoder_lora_state_dict.pop(f"{name}.to_out_lora.down.weight")
 
+                if text_encoder_lora_state_dict:
+                    for name, _ in text_encoder_aux_modules(text_encoder):
```

Comment:
Would it make sense to do the iteration if "aux" module keys are detected in the state dict?